us.zoom.Zoom.json

```diff
@@ -12,8 +12,10 @@
         "--share=ipc",
         "--socket=x11",
         "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
This gives full access to the user's dbus, no? This is quite a biggie.
I don't feel comfortable giving away access to the full dbus without any justification.
Do you know what it's talking to? Those paths can be given access to.

Reply:
fair, there's no real need to give it access to the session